src/_shared/scss/_adverts-masterclasses.scss

```diff
@@ -1,42 +1,48 @@
+
+.adverts--tone-masterclasss .button--large {
+    background: #ffffff;
+    color: #000000;
+}
+
 .adverts--masterclasss {
 }
 
 .adverts--tone-masterclasss {
     > .adverts__header {
-        background: $multimedia-main-1;
+        background: $guardian-masterclasses-rebrand;
         .button {
-            color: $multimedia-main-2;
-            border-color: fade-out($multimedia-main-2, .7);
+            color: #000000;
+            border-color: fade-out($guardian-masterclasses-rebrand, .7);
 
             &:hover,
             &:focus,
             &:active {
-                border-color: $multimedia-main-2;
+                border-color: $guardian-masterclasses-rebrand;
             }
         }
     }
 
     .adverts__logo {
-        color: $multimedia-main-2;
+        color: $guardian-masterclasses-rebrand;
     }
 
     .inline-commercial-brand svg {
-        fill: $multimedia-main-2;
+        fill: $guardian-masterclasses-rebrand;
     }
 }
 
 .adverts--tone-masterclasss .button--legacy-single,
 .advert--masterclass .button {
-    background: $multimedia-main-2;
-    color: $neutral-1;
+    background: $guardian-masterclasses-rebrand;
+    color: #000000;
```

Comment:
Same as above - it seems that all the buttons are `color: #000000` - perhaps we can factor this out.